src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetHash.cs

```diff
@@ -156,6 +156,16 @@ protected override void ProcessRecord()
                         path);
                     WriteError(errorRecord);
                 }
+                catch (IOException ioException)
+                {
+                    ThrowTerminatingError(
```

Comment:
This should be WriteError() too - as in the original WinPS implementation, any exception for a given pipeline input should be a non-terminating error.

Reply:
Yes, It is was copy-past without fix.